includes/FeatureAPI/Features/Settings/SettingsManager.php

```
@@ -0,0 +1,668 @@
+				'categories' => [ 'dokan', 'settings', 'marketplace' ],
+			]
+        );
+    }
+
+    /**
+     * Get general settings
+     *
+     * @param array $context
+     * @return array
+     */
+    public function get_general_settings( $context ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$context'. (Unused Code Rules)

(UnusedFormalParameter)